ui/app/pages/confirm-add-suggested-token/confirm-add-suggested-token.component.js

```diff
@@ -45,6 +48,20 @@ export default class ConfirmAddSuggestedToken extends Component {
           <div className="page-container__subtitle">
             { this.context.t('likeToAddTokens') }
           </div>
+          { hasTokenDuplicates ?
+            (
+              <div className="warning">
+              { this.context.t('knownTokenWarning') }
```

Comment:
The last two lint errors are fixed, but this line and line 61 need to be indented one more time, as they're inside the `div` block

This should all be caught by the `yarn lint` command you can run locally as well btw

Reply:
The awful thing is for some reason I can't even run `lint` locally or I'll run out of memory. That might be enough reason to upgrade comps. Or maybe just push to a cloud-hosted linting machine first..

Reply:
Wow, that's unfortunate. Maybe it would work if you reduced the memory Node was allowed to use? You can do that via an environment variable: `NODE_OPTIONS=--max_old_space_size=[amount of memory in MB]`